dist/index.html

```diff
@@ -14,12 +14,42 @@
     <h1 class='top-nav'></h1>
     <input class='search-destinations' placeholder='Search Locations'></input>
     <button class='search-button'>Search  </button>
-    <button class='create-new-trip'>New Trip</button>
+    <button class='create-new-trip trigger'>New Trip</button>
   </nav>
   <main>
+    <div class='modal'>
+      <span class="close-button">X</span>
+      <div class='modal-content'>
+        <form class='new-trip' action='create-new-trip-form' method='post'>
```

Comment:
I didn't know you could assign a method in a form! Had to look that one up. That's cool!